src/dlr_relayvm.cc

```diff
@@ -278,7 +279,11 @@ void RelayVMModel::GetOutputSizeDim(int index, int64_t* size, int* dim) {
   *size = 1;
   if (index < outputs_.size()) {
     auto arr = outputs_[index];
-    *size = std::accumulate(arr->shape, arr->shape + arr->ndim, 1, std::multiplies<int64_t>());
+    if (dlr::HasNegative(arr -> shape, arr -> ndim)) {
+      *size = -1;
```

Comment:
This could be an error code like
```
BAD_SIZE_ERR
```

Reply:
-1 is a valid size for RelayVM models, this PR is just so GetOutputSizeDim returns a consistent value for an unknown size

Reply:
I misunderstood your response:
All I was saying is:
```c++
constexpr int UNKNOWN_SIZE = -1;
```